### PR TITLE
Fix broken EventAggregator link

### PR DESCRIPTION
**Pull request by KrzysztofKozlowskiOBSS**

### CODE REVIEW

File: markdown/en/core-result-dispatcher.md [hunk 1]
```diff
@@ -4,7 +4,7 @@ subscribe to ResultDispatcher to get notified when new data is available.
 
 The ResultDispatcher is in the module: **factfinder.communication.ResultDispatcher**.
 
-If you want to trigger your own communication events checkout the documentation for the [FFCommunicationEventAggregator](documentation/core-event-aggregator).
+If you want to trigger your own communication events checkout the documentation for the [FFCommunicationEventAggregator](api/core-event-aggregator).
```
Comment (omikron-fp) [inline markdown/en/core-result-dispatcher.md]: I would make it `api/core-event-aggregator#tab=docs` to the link. When the fragment is added by the app, another entry in the browser history is created and you have to click twice to navigate back. (At least in my Chrome)

**Comment by KrzysztofKozlowskiOBSS** [comment]
@omikron-fp can you check if we can remove the history entry without hash param with #tab=docs instead of creating a new one?

**Comment by omikron-fp** [comment]
@KrzysztofKozlowskiOBSS Sure, that would actually be the more desirable behaviour. Might take me a little to dive into the code.

**Comment by KrzysztofKozlowskiOBSS** [comment]
@omikron-fp https://github.com/FACT-Finder-Web-Components/documentation/blob/2234b30df9c6e236676c7c8cb0c26ddae30a4fd6/src/views/api-view.js#L198-L199 if you manage to replace the location hash here manually, our router shouldn't add a new history entry

**Pull request merged**